classes/Bloodline.php

```diff
@@ -15,6 +15,8 @@ class Bloodline {
 
     const BOOST_POWER_PRECISION = 5;
 
+    const HEAL_RANGE_PERCENT = 25;
```

Comment:
Could you add a comment explaining this? (healing varies from -HEAL_RANGE_PERCENT at 100% health, to +HEAL_RANGE_PERCENT at 0% health)

Reply:
75% effectiveness at 100% HP, 125% effectiveness at 0% HP

Reply:
Reverted Heal to original mechanics.